src/redux/reducers/interests.js

```diff
@@ -0,0 +1,12 @@
+import CNST from "../../constants";
+
+import interests from "../stores/interests";
+
+export default function reducer(state = interests, action) {
```

Comment:
I think it is not a good idea to create a separate reducer for interests.
I suggest creating one reducer for sign up, because when we decide to submit everything it will be easy to get data from store. Let me know if it is not clear